owncloudApp/src/main/java/com/owncloud/android/utils/MdmConfigurations.kt

```diff
@@ -26,12 +26,14 @@ const val CONFIGURATION_LOCK_DELAY_TIME = "lock_delay_time_configuration"
 const val CONFIGURATION_SERVER_URL = "server_url_configuration"
 const val CONFIGURATION_SERVER_URL_INPUT_VISIBILITY = "server_url_input_visibility_configuration"
 const val CONFIGURATION_ALLOW_SCREENSHOTS = "allow_screenshots_configuration"
+const val CONFIGURATION_WEBFINGER_LOOKUP_SERVER = "webfinger_lookup_server_configuration"
```

Comment:
Not sure if this fits well here since it doesn't have to do with MDM...

Reply:
Let's use the `NO_MDM_RESTRICTION_YET` 👍